participant/playwright.config.ts

```diff
@@ -30,7 +30,7 @@ const config: PlaywrightTestConfig = {
        * a specific command:
        * https://github.com/basarat/demo-playwright-vrt/blob/main/.github/workflows/update-snapshots.yml
        */
-      maxDiffPixelRatio: 0.01,
+      maxDiffPixelRatio: 0,
```

Comment:
You're going to have a bad time, since we're copying the Darwin snapshots generated by devs into Linux snapshots for running in Github
(and, i spent SOME time trying to get the github action to run in Macos, but apparently licensing for Docker means they don't install Docker on Github Workflow images..)

Reply:
I'd rather rely on Chromatic for copy change flags, but 🤷🏻‍♀️

Reply:
> You're going to have a bad time, since we're copying the Darwin snapshots generated by devs into Linux snapshots for running in Github (and, i spent SOME time trying to get the github action to run in Macos, but apparently licensing for Docker means they don't install Docker on Github Workflow images..)

Oh right. Thank you. I couldn't remember exactly why we made the `maxDiffPixelRatio` not 0. I'm ok with relying on Chromatic for copy changes.

Reply:
> You're going to have a bad time, since we're copying the Darwin snapshots generated by devs into Linux snapshots for running in Github (and, i spent SOME time trying to get the github action to run in Macos, but apparently licensing for Docker means they don't install Docker on Github Workflow images..)

Oh right. Thank you. I couldn't remember exactly why we made the `maxDiffPixelRatio` not 0. I'm ok with relying on Chromatic for copy changes.